src/wh_client_cryptocb.c

```diff
@@ -90,10 +94,12 @@ int wolfHSM_CryptoCb(int devId, wc_CryptoInfo* info, void* inCtx)
     {
     case WC_ALGO_TYPE_CIPHER:
         /* set type */
+#ifdef HAVE_AES
         packet->cipherAnyReq.type = info->cipher.type;
         /* set enc */
         packet->cipherAnyReq.enc = info->cipher.enc;
         switch (info->cipher.type)
```

Comment:
you should not be macroing out the switch construct like this, it make it difficult to read. Just protect the calls to the wolfCrypt APIs

Reply:
What cipher options are there other than AES, because when no AES is defined cipher will not work. So should I be worrying about this switch statement not working when AES isn't defined?

Reply:
wolfCrypt has other ciphers that it supports. That said, I'm actually realizing that we don't need these macros here, as this cryptoCallback is invoked internally by wolfCrypt functions, which will not actually be compiled if they have built with `NO_AES` and friends. So since these are just sending requests to the server, we might be A-OK leaving this file as is. Let me discuss this with some other folks and I'll get back to you

Reply:
Yeah @jackctj117 you don't need to add protection to this file.

Reply:
Ok sounds good I'll remove it from the PR.